### PR TITLE
Adding check URLs to GH actions

### PR DESCRIPTION
**Pull request by dwertent**
I've noticed some invalid URLs in the repository. This GitHub Action will help detect these invalid URLs when opening a pull request or during a scheduled scan. I recommend setting `force_pass` to `false` to prevent the approval of pull requests containing invalid URLs. You can see an example [here](https://github.com/dwertent/firefly/actions/runs/9275582818/job/25520636370#step:4:351).

image

### CODE REVIEW

**Comment by dwertent** [comment]
Working on this PR made me realize that the package should support SARIF format output. As a result, I opened [this PR](https://github.com/urlstechie/urlchecker-python/pull/91) and [this PR](https://github.com/urlstechie/urlchecker-action/pull/109) to enhance the informativeness of the GitHub Action results.

**Review by EnriqueL8 — CHANGES_REQUESTED** [review]
Thanks for the contribution here

I'm torn on the force pass always being `true` as we won't notice if thing have broken without looking at logs but at the same time I don't want this to gate PRs into the repo 

So if there is a way to be notified that it has failed on the schedule it would be good.

As part of this PR would be a great contribution if the existing URL highlighted in the description are fixed as well

File: .github/workflows/url_checker.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v3
+    - name: Run URLChecker
+      uses: urlstechie/urlchecker-action@0.0.34
+      with:
+        file_types: .md
+        force_pass: true
+        save: results.csv
+        exclude_patterns: 127.0.0.1,localhost,ipfs_,ca_org,ethconnect_,dataexchange_
```
Comment (EnriqueL8) [inline .github/workflows/url_checker.yml:20]: Also add `tokens_0_0` here

**Comment by EnriqueL8** [comment]
I can't edit my review seems like you put in your description `I recommend setting force_pass to false to prevent the approval of pull requests containing invalid URLs.` that is valid to me then

**Pull request closed without merging**